src/components/dropdown/dropdown.spec.js

```diff
@@ -96,20 +96,6 @@ describe('dropdown', async () => {
     expect(Array.from(dd_6.$refs.menu.children).find(node => node.innerHTML === 'button')).toBeElement('button')
   })
 
-  it('dd-item-button should emit click event', async () => {
```

Comment:
Actually the dropdown-item-button should emit a `clicked::link' event (which is fired after any user`click` handlers are run, as the event is merged in last after any user supplied events), just like `b-link` fires the `clicked::link` event to  lose the dropdown.